Cannot build on windows without commenting out line 154+ in CMakeLists.txt

When pthread.h is not available (didn't test with an available one), the build fails with:

CMake Error at CMakeLists.txt:158 (set_target_properties):
  set_target_properties Can not find target to add properties to:
  Threads::Threads

--- a/liquidfun/Box2D/CMakeLists.txt
+++ b/liquidfun/Box2D/CMakeLists.txt
@@ -37,6 +37,10 @@ if(CMAKE_CXX_COMPILER MATCHES ".*clang")
   # set_target_properties Can not find target to add properties to: Threads::Threads
   find_package(Threads)
 endif()
+if (WIN32)
+  find_package(Threads)
+endif()
+
 
 add_definitions( -DLIQUIDFUN_EXTERNAL_LANGUAGE_API=1 )
 

